static/src/javascripts/projects/commercial/modules/mobile-sticky.js

```diff
@@ -34,7 +34,7 @@ export const init = (): Promise<void> => {
                 const mobileStickyAdSlot = mobileStickyWrapper.querySelector(
                     '#dfp-ad--mobile-sticky'
                 );
-                if (mobileStickyAdSlot) addSlot(mobileStickyAdSlot, false);
+                if (mobileStickyAdSlot) addSlot(mobileStickyAdSlot, true);
```

Comment:
What's the significance of switching the 2nd argument from  `false` to `true`?

Reply:
It is switching the `forceDisplay` flag boolean which eventually decides if the slot should be lazy-loaded and eagerly-loaded. It was found that mobile sticky was not appearing always immediately due to the lazy-load behaviour. Switching to true(eager load)  optimises the performance of the mobile sticky to be always loaded